index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="es">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Curriculum Vitae</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Bruno Perez Merino</h1>
+    </header>
+    <main>
+      <h3>Perfil</h3>
+      <p>
+        Estudiante de Ingeniería Informática con alto rendimiento académico y
+        experiencia en apoyo académico a compañeros universitarios.
+        <br />Destacado por su liderazgo y compromiso en la enseñanza de
+        materias complejas, con sólidos conocimientos en programación.
+      </p>
```

Comment:
Para evitar usar saltos de línea "br" mi recomendación es partir este párrafo en dos párrafos. Así logra el efecto deseado y es más fácil de mantener.

```suggestion
      <p>
        Estudiante de Ingeniería Informática con alto rendimiento académico y
        experiencia en apoyo académico a compañeros universitarios.
        </p>
        <p>Destacado por su liderazgo y compromiso en la enseñanza de
        materias complejas, con sólidos conocimientos en programación.
      </p>
```